include/fluidsynth.h

```diff
@@ -48,7 +48,7 @@ extern "C" {
 
 #if defined(__GNUC__) || defined(__clang__)
 #    define FLUID_DEPRECATED __attribute__((deprecated))
-#elif defined(_MSC_VER)
+#elif defined(_MSC_VER) && _MSC_VER > 1400
```

Comment:
Perhaps `>= 1500` ?

Reply:
Unfortunately, I cannot verify it.
MSVC surely does not support it.

Reply:
After some research, I think that MSVC6 was really the last version to not support deprecated declspec, so I modified the test on _MSC_VER to be more strict.